src/window/city.c

```diff
@@ -512,6 +513,14 @@ static void handle_hotkeys(const hotkeys *h)
             }
         }
     }
+    if (h->storage_order) {
+        int building_id = map_building_at(widget_city_current_grid_offset());
+        int grid_offset = widget_city_current_grid_offset();
```

Comment:
Why not invert the lines instead of calling `widget_city_current_grid_offset()` twice?

Reply:
because of...idk :D
you are right, thanks!